recipes/underworld/meta.yaml

```
@@ -0,0 +1,74 @@
+{% set name = "underworld" %}
+{% set version = "2.14.2" %}
+{% set sha256 = "42a806a60f2ed37957e92e7fd6974a3ab292fcb80eafc64c4f64f993181fcb8b" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/underworldcode/underworld2/archive/refs/heads/v2.14.x.zip
+  sha256: {{ sha256 }}
```

The version should be a jinja variable here so the auto-update bot works. But you will need to provide one that is PEP 440 compliant. that `x` at the end isn't.

```suggestion
{% set name = "underworld" %}
{% set version = "2.14.x" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://github.com/underworldcode/underworld2/archive/refs/heads/v{{ version }}.zip
  sha256: 42a806a60f2ed37957e92e7fd6974a3ab292fcb80eafc64c4f64f993181fcb8b
```

---

Updated - thanks for pointing it out.